.platform/hooks/predeploy/05_set_git_profile.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Get the env type from EB
+ENV_TYPE=$(/opt/elasticbeanstalk/bin/get-config environment -k SSM_PREFIX)
+
+GIT_USER_NAME_PARAM_NAME="${ENV_TYPE}_GIT_USER_NAME"
+GIT_USER_EMAIL_PARAM_NAME="${ENV_TYPE}_GIT_USER_EMAIL"
+
+# Set AWS region
+aws configure set default.region ap-southeast-1
+
+# Fetch git user's name and email from SSM
+GIT_USER_NAME=$(aws ssm get-parameter --name $GIT_USER_NAME_PARAM_NAME --with-decryption --query "Parameter.Value" --output text)
+GIT_USER_EMAIL=$(aws ssm get-parameter --name $GIT_USER_EMAIL_PARAM_NAME --with-decryption --query "Parameter.Value" --output text)
+
+# Write the configuration to .gitconfig
+echo "[user]" >> /home/webapp/.gitconfig
+echo "  name = $GIT_USER_NAME" >> /home/webapp/.gitconfig
+echo "  email = $GIT_USER_EMAIL" >> /home/webapp/.gitconfig
```

Comment:
Separately, if we run this multiple times, it will keep appending the 3 lines to `.gitconfig`. Is it better to check if they already exist before appending them? 
Or maybe overwriting every time we rung the script?

Reply:
good point, I added the overwrite